Themis/Views/Exercises/ExerciseSection.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  ExerciseSection.swift
+//  Themis
+//
+//  Created by Tom Rudnick on 12.11.22.
+//
+
+import SwiftUI
+import SharedModels
+
+struct ExerciseDateProperty: Hashable {
+    let name: String
+    let dateKeyPath: KeyPath<Exercise, Date?>
+}
+
+struct ExerciseSection: View {
+    @EnvironmentObject var courseVM: CourseViewModel
+    
+    var exercises: [Exercise]
+    
+    private let dueDate = ExerciseDateProperty(name: "Submission", dateKeyPath: \.baseExercise.dueDate)
+    private let assessmentDueDate = ExerciseDateProperty(name: "Assessment", dateKeyPath: \.baseExercise.assessmentDueDate)
+    private let releaseDate = ExerciseDateProperty(name: "Release", dateKeyPath: \.baseExercise.releaseDate)
```

Comment:
May change the order according to the chronological appearance.

```suggestion
    private let releaseDate = ExerciseDateProperty(name: "Release", dateKeyPath: \.baseExercise.releaseDate)
    private let dueDate = ExerciseDateProperty(name: "Submission", dateKeyPath: \.baseExercise.dueDate)
    private let assessmentDueDate = ExerciseDateProperty(name: "Assessment", dateKeyPath: \.baseExercise.assessmentDueDate)
```